### PR TITLE
feature-NXP-21488-refactor-nuxeo-document-page

### PR DESCRIPTION
**Pull request by adilioaraujo**

### CODE REVIEW

**Comment by Gabez0r** [comment]
👍

File: elements/document/nuxeo-document-page.html [hunk 1]
```diff
@@ -135,83 +69,35 @@
...
-
-          <!-- activity -->
-          <paper-card>
-            <h3>[[i18n('documentPage.activity')]]</h3>
-            <nuxeo-document-activity document="[[document]]"></nuxeo-document-activity>
-          </paper-card>
-
-        </template>
+        <!-- info -->
+        <paper-card hidden$="[[isProcessingTask]]">
```
Comment (nelsonsilva) [inline elements/document/nuxeo-document-page.html]: thought you were a fan of the paper-card mixin ;)

File: elements/document/nuxeo-document-page.html [hunk 2]
```diff
@@ -135,83 +69,35 @@
...
+          <div hidden$="[[edit]]">
+            <paper-icon-button id="edit" icon="editor:mode-edit" on-tap="_editMode" hidden$="[[!_canEdit(document)]]"></paper-icon-button>
+            <nuxeo-document-metadata document="[[document]]"></nuxeo-document-metadata>
+          </div>
+          <nuxeo-document-edit document="[[document]]" hidden$="[[!edit]]"></nuxeo-document-edit>
+        </paper-card>
+
+        <!-- workflows -->
+        <nuxeo-document-workflows document="[[document]]" is-processing-task$="[[isProcessingTask]]"></nuxeo-document-workflows>
```
Comment (nelsonsilva) [inline elements/document/nuxeo-document-page.html]: idea was to move most of the workflow logics to this element and then do `is-processing-task$={{isProcessingTask}}`. I guess this way we can remove at least some listeners from here

File: elements/document/nuxeo-document-page.html (hunk 1 above)
Comment (nelsonsilva) [inline elements/document/nuxeo-document-page.html]: did someone confirm we want to hide all of these while processing a task?!

File: elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<!--
+(C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
```
Comment (nelsonsilva) [inline elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html]: (C) Copyright ~~2016~~2017 Nuxeo ~~SA~~ (http://nuxeo.com/) and others.

File: elements/workflow/nuxeo-document-workflows.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
+<!--
+(C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
```
Comment (nelsonsilva) [inline elements/workflow/nuxeo-document-workflows.html]: see ☝️

File: elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html (hunk 3 above)
Comment (adilioaraujo) [inline elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html]: :+1:

File: elements/workflow/nuxeo-document-workflows.html (hunk 4 above)
Comment (adilioaraujo) [inline elements/workflow/nuxeo-document-workflows.html]: :+1:

**Pull request merged**